### ISSUE TITLE
[Rule] *** Tests / Tasks *** should not be defined in resource file

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#524]
Right now we are recognizing file type in this code: 
https://github.com/MarketSquare/robotframework-robocop/blob/af807515d7f160f6673f726f3b7cd33bca092a21/robocop/run.py#L93

(basically, we check if the file imports something, and if we're importing file that's being scanned by us - it's set as resource type). 

This approach have some drawbacks, namely we cannot recognize resource files if we're not importing them in any scanned file. For example:
```
robocop file.resource
```
It will never be imported by other file (because we're only scanning one). 

We have also proposed renaming files without `*** Tests ***` / `*** Tasks *** ` to `.resource` extension with newly added "can-be-resource-file". 

I propose to add bit of reverse rule - if the file have `.resource` extension and is not recognized as resource file by our method, we should report if it contains  `*** Tests ***` / `*** Tasks *** ` sections.

**Comment by mnojek** [comment]
@bhirsz I think this is a duplicate of #525. They were created on the same day, but I can't understand what's the difference between them. Maybe you remember?

**Comment by bhirsz** [comment]
I think the issue is that we already report this as parsing-error because Robot Framework detect it. If we want separate rule we can just catch it and raise as different rule. But we only need the other issue, not this one

**Comment by mnojek** [comment]
> But we only need the other issue, not this one

You mean that we need the issue from #525, and this can be closed?

**Comment by bhirsz** [comment]
Yes, exactly. Closing this issue
